test/fixtures/go/corpus/assignment-statements.diffA-B.txt

```diff
@@ -8,12 +8,7 @@
       { (Identifier)
       ->(Identifier) }
         (Integer))
-      (Assignment
-        (Statements
-        { (Identifier)
-        ->(Identifier) }
-        { (Identifier)
-        ->(Identifier) })
+      (AugmentedAssignment
```

Comment:
Various changes to shape of assignment trees due to things like adding `AugmentedAssignment`.